tools/regression_test.sh

```diff
@@ -137,11 +137,12 @@ function main {
   fi
   if [ $TEST_MODE -ge 1 ]; then
       build_checkpoint
-      run_db_bench "readrandom"
-      run_db_bench "readwhilewriting"
-      run_db_bench "deleterandom" $((NUM_KEYS / 10 / $NUM_THREADS))
-      run_db_bench "seekrandom"
-      run_db_bench "seekrandomwhilewriting"
+      #run_db_bench "readrandom"
+      #run_db_bench "readwhilewriting"
+      #run_db_bench "deleterandom" $((NUM_KEYS / 10 / $NUM_THREADS))
+      #run_db_bench "seekrandom"
+      #run_db_bench "seekrandomwhilewriting"
+      run_db_bench "multireadrandom" 
```

Comment:
The multireadrandom already covers all the benchmarks being removed here?

Reply:
Oh sorry! Had commented them out for testing purposes.